### ISSUE TITLE
Page excessively caching

### ISSUE THREAD

**Issue by KindaOK** [michiganhackers/mh-web-new#95]
It seems like the issue is not the browser cache, but may be related to the CRA service worker. Although that is failing to register so ¯\\_(ツ)_/¯. What is known is that the first navigation to a page directly directly goes to the cache and thus ignores updates. Possibilities are adding a query param to force the reload, getting the last updated date from s3 (ping a text file with last update date) and then location.reload, or something else.

**Comment by KindaOK** [comment]
So we've tried a few things. One is adding a Cloudfront behavior that disables caching for `index.html`. This seems to only apply at the CDN level and not for browsers [0]. Another property that we've set is to add a `cache-control` header via the metadata tags to the `index.html` s3 object [1]. We'll see if these work how we want them to. Barring that, we may look at the service worker approach. I would rather not use it however because we don't need it for a simple static site.

[0] https://stackoverflow.com/a/54462509
[1] https://stackoverflow.com/a/51275192
